### PR TITLE
Gb/qa

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

File: sup3r/pipeline/forward_pass.py [hunk 1]
```diff
@@ -46,9 +44,9 @@ class ForwardPassStrategy(InputMixIn):
 
     def __init__(self, file_paths, model_args, s_enhance, t_enhance,
                  fwp_chunk_shape, spatial_pad, temporal_pad,
+                 temporal_slice=slice(None),
```
Comment (bnb32) [inline sup3r/pipeline/forward_pass.py:47]: while we're editing this could we add the hr_spatial_coarsen arg to pass to the data handler?

Comment (grantbuster) [reply sup3r/pipeline/forward_pass.py:47]: Yeah can do, whats the use case? Would this be for doing validation with data thats already high res?

Comment (bnb32) [reply sup3r/pipeline/forward_pass.py:47]: Yeah that exactly.

**Review by bnb32 — APPROVED** [review]
Very clean. Looks great.

File: sup3r/pipeline/forward_pass.py (hunk 1 above)
Comment (grantbuster) [reply sup3r/pipeline/forward_pass.py:47]: cool, done!

**Pull request merged**